plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveMetadataListing.java

```diff
@@ -106,6 +106,45 @@ public class TestHiveMetadataListing
             Optional.of("SELECT 1"),
             OptionalLong.empty());
 
+    private static final Table CORRECT_TABLE = new Table(
+            DATABASE_NAME,
+            "correct_table",
+            Optional.of("owner"),
+            "table_type",
+            TABLE_STORAGE,
+            ImmutableList.of(TABLE_COLUMN),
+            ImmutableList.of(TABLE_COLUMN),
+            ImmutableMap.of("param", "value3"),
+            Optional.of("original_text"),
+            Optional.of("expanded_text"),
```

Comment:
These arguments can be `Optional.empty()` as this constant is a table. Same for others.